wiki/Tournaments/MCNC/2019/en.md

```diff
@@ -0,0 +1,354 @@
+
+- ModType
+  - [Frums - Wavetapper (qqqant) [Stage 1: Memory Factory]](https://osu.ppy.sh/b/1811863)
+  - [stereoberry - evangelize (blurry images) (Tidek) [transparency (SV)]](https://osu.ppy.sh/b/1222765)
+  - [Beltaine - Rockhill (Ryu Sei) [Hard]](https://osu.ppy.sh/b/994291)
+  - [PROTODOME - Greatest Hat (Gekido-) [chalLeNge]](https://osu.ppy.sh/b/1759313)
+  - [Frederic - oddloop (infilect) [oldnep]](https://osu.ppy.sh/b/1376904)
+  - [eye burn - SHI*A*WA*SE (Elekton) [joy]](https://osu.ppy.sh/b/1552444)
+  - [wa. remixed by celas - Gin no Kaze (Elekton) [mistral]](https://osu.ppy.sh/b/1592941)
+  - [Silentroom - Protoflicker ([Crz]Crysarlene) [crysather]](https://osu.ppy.sh/b/1954837)
+  - [Aural Vampire - Shonan Zoku -Cannibal Coast- (Wh1teh) [k]](https://osu.ppy.sh/b/1158510)
+  - [The Flashbulb - Chik Habit (Gekido-) [Stage 4: Torment]](https://osu.ppy.sh/b/1546411)
+  - [Laur, Dustvoxx - FireLight (Neokontrol Remix) (Suu is my waifu) [Enkindle]](https://osu.ppy.sh/b/1708743)
+  - [Soko ni Naru - Tenohira de Odoru (Vortex-) [Sweaty Palms]](https://osu.ppy.sh/b/1711260)
+  - [Camellia - *Feels Seasickness...* (Suu is my waifu) [*Maximum...*]](https://osu.ppy.sh/b/1932640)
+- Tiebreaker
+  - **[goreshit - fleshbound ([Crz]NucleoAP) [Couil's trapped]](https://osu.ppy.sh/b/2075130)**
+
+### Group Stage
+
+**[Download the mappack here! (84.3MB)](https://www.dropbox.com/s/ywwvekemymwzrjh/MCNC19%20%E5%B0%8F%E7%BB%84%E8%B5%9B%E5%9B%BE%E5%8C%85.rar?dl=0)**
+
+- Freemod
+  - [Frums - Credits (BossModWolf) [Glitched Tech Corp. (SV)]](https://osu.ppy.sh/b/1985858)
+  - [succducc - me & u (Couil) [& sv]](https://osu.ppy.sh/b/1442650)
+  - [FELT - Life [Eris's "a Fate Encounter" mix] ([OSC]Amagi) [Lunatic]](https://osu.ppy.sh/b/1971029)
+  - [Draw the Emotional & Foreground Eclipse - Stay by my side (Herkkupala) [puxtu's Lunatic]](https://osu.ppy.sh/b/968811)
+  - [ARM - Saita, Kimi no Egao (Julie) [I can Fly]](https://osu.ppy.sh/b/967569)
+  - [Various Artists - Chordial Blitz (DourGent) [Venetian Drums - Sheenoboo]](https://osu.ppy.sh/b/1703314)
+  - [Joywave - Yolomania Selections (Daybreak) [1.1x]](https://osu.ppy.sh/b/2066048)
+  - [BilliumMoto - life flashes before weeb eyes (Hydria) [Tech]](https://osu.ppy.sh/b/2041466)
+  -	[void - Just Hold On (LovelySerenade) [Flawless Diversion (challenge)]](https://osu.ppy.sh/b/1467814)
+  - [Rengoku Teien - Suikyou Heaven (Kamikaze) [Oriental // justhow x Kamikaze]](https://osu.ppy.sh/b/1755454)
+  - [Mick Gordon - Rip and tear (Herkkupala) [Revenant]](https://osu.ppy.sh/b/1970086)
+- Tiebreaker
+  - **[Kano - Stella-rium (DJ Noriken Bootleg) (Lude) [Koneko's Celestial // 4K]](https://osu.ppy.sh/b/1253374)**
+
+
+--------------------------------
+
+## Match results
+
+### Finals
+
+| Blue team | | | Red team | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **[Crz]Murasame**  | **7** | 3 |  [Crz]FolAH1217 | [#1](https://osu.ppy.sh/community/matches/54022298) |
```

Comment:
```suggestion
| **\[Crz\]Murasame** | **7** | 3 | \[Crz\]FolAH1217 | [#1](https://osu.ppy.sh/community/matches/54022298) |
```
You also only need 1 cell padding for filled content.
And 2 cell padding if that's empty.
Apply for all.